README.md

```diff
@@ -59,11 +59,39 @@ pip install -r requirements.txt
 
 We're using [this library](https://aiohttp.readthedocs.io/en/stable/) to improve speed of polling the TTC's API by making requests asynchronous. Installation was... fine in Ubuntu 16.04 and OSX, had some hiccoughs in Debian/Raspbian. Stay tuned.
 
-### Database setup
+## The Scraper
+We can retrieve the state of the subway via the `ttc_scraper_api.py` script.
 
-The database engine used to store the data is PostgreSQL, you can find instructions to get the latest and greatest version [here](https://www.postgresql.org/). After you've set up your database you can run the contents of `create_tables.sql` in a pgAdmin query window (or run it as a sql query). 
+This will fetch the arrival information for each subway station and persist it. The options to 
+this are specified in the help, it needs an action as well as  a destination 
+(e.g. scrape and --postgres).
 
-#### Edit `db.cfg`
+### Data Destinations
+The scraper script is able to persist the results to either Postgres, or AWS S3. One (and only one)
+of these must be chosen with either the --postgres or the --s3 flag.
+
+#### S3
+If the --s3 flag is set then the S3_BUCKET environmental variable must be set which specifies 
```

Comment:
Trying to run through this documentation.
Ok so I managed to create a bucket for my AWS account (way to have wayyyyy too many options AWS) and I'm confused as to what this bucket variable should look like.

Reply:
sorry! I will update this, basically its just the name of the bucket you created to put the scrapes in, as listed in the aws console

![image](https://user-images.githubusercontent.com/3654730/50224279-76bd1d00-036b-11e9-8b2d-325aa0f48363.png)

in my particular case it would be "ttc.scrape" - AWS buckets are global so nobody else can pick that one.

Reply:
btw the "ttc-subway-times-dev-serverless..." bucket was created by serverless to hold the scraper code as that is where AWS lambda reads from.